client/cli/src/commands/inspect_key.rs

```diff
@@ -28,7 +28,7 @@ use structopt::StructOpt;
 	name = "inspect-key",
```

Comment:
Just noticed that this should have been changed?

Reply:
```suggestion
	name = "inspect",
```